index.md

```diff
@@ -37,6 +37,12 @@ There are several private [Facebook Groups with super-friendly and helpful Arlin
 
 I'm [actively looking for collaborators](https://menotomymatters.com/howto/covid-organize/) to help turn this site into a master jump-off point to better publicise the many different volunteer groups and efforts in Arlington.  Read about my [ideas for this site over on MenotomyMatters](https://menotomymatters.com/howto/covid-organize/), my Arlington-focused blog.
 
+### Technical Assistance
```

Comment:
Please update your line to note briefly **which** services (Zoom, hangouts, wifi, whatever) you could help someone with and resubmit.
Note that in the future, services listings like this will be in a sub-page, likely driven from a CSV file or the like.